### ISSUE TITLE
Scan feature

### ISSUE THREAD

**Issue by overload08** [AHAAAAAAA/PokemonGo-Map#2048]
Hi all.

I have larges area to scan (city in one scan).

I think that's will be cool to have command switch to relauch a new scan every X minutes or seconds.

Exemple : My map is scanned in 6 minutes. If the switch is set to 1 minute, the script will run 6 scans with 1 minute delay.

I don't know if I'm clear...

**Comment by tplkV2** [comment]
It already starts scan over again after finishing.

**Comment by overload08** [comment]
How to do ?

**Comment by overload08** [comment]
Oops, Yes after finish, but this feature launch a new scan at the same time with previous scan.

With that, i can scan a location every X minute rather than wait 6 minutes...

**Comment by tplkV2** [comment]
Oh, I got it.
Well, right now you can manually launch new instance each minute.

**Comment by jforr** [comment]
It sounds like the issue is that very large scans do not complete in an adequate amount of time.  It seems like the solution would be to switch to the mysql option and break your scan into several smaller scans.

**Comment by overload08** [comment]
I thought about smaller area but it's more complex... relaunch instance too...

**Comment by Sejadis** [comment]
its more complex yes but it isnt that complex to run multiple smaller areas
i.e. have 13 processes running which i all start by running 1 batch file and that batch file also handles restarting all processes every hour

**Comment by overload08** [comment]
I'll try

**Comment by DeviantEng** [comment]
Yeah, I think a `-ss (single scan)` switch would be neat.  I currently am scanning 5 small (rural-ish) cities in eastern Kentucky, and I have to take the time to break up several searches based on coordinates.  Not hard; just time consuming.  Each has 7 steps, 2 threads, and I probably have about 30 total scans running.  Having a single scan option would allow me to set location based on the city with a large radius, and kick off a new scan every 60 seconds.  No more dealing with aligning coordinates with slight overlap.
